src/components/Book.js

```diff
@@ -13,12 +15,11 @@ const Book = ({ title, author, id }) => {
   };
 
   return (
-    <div>
-      <h2>{title}</h2>
-      <h3>{author}</h3>
-      id:
-      {id}
-      <button type="button" onClick={deleteBook} value={id}>Delete</button>
+    <div className="book font2">
+      <text className="category font1">{category}</text>
+      <h2 className="title">{title}</h2>
+      <h3 className="author">{author}</h3>
+      <button className="delete" type="button" onClick={deleteBook} value={id}>Delete</button>
```

Comment:
- [ ] The design looks a lot different from yours, kindly updated it according to your design

YOurs: ![image](https://user-images.githubusercontent.com/62705947/191988406-e9cb0774-bf84-4e67-99ee-62e360383a4b.png)

Design:![image](https://user-images.githubusercontent.com/62705947/191988481-92cad623-5213-4b0f-9179-289efad76109.png)